### ISSUE TITLE
[Accepted] SDL 0228 - SDL Server Email Notifications

### ISSUE THREAD

**Issue by theresalech** [smartdevicelink/sdl_evolution#721]
Hello SDL community,

The review of " SDL 0228 - SDL Server Email Notifications" begins now and runs through May 14, 2019.  The proposal is available here:

https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0228-sdl-server-email-notifications.md

Reviews are an important part of the SDL evolution process. All reviews should be sent to the associated Github issue at:

https://github.com/smartdevicelink/sdl_evolution/issues/721

What goes into a review?

The goal of the review process is to improve the proposal under review through constructive criticism and, eventually, determine the direction of SDL. When writing your review, here are some questions you might want to answer in your review:

* Is the problem being addressed significant enough to warrant a change to SDL?
* Does this proposal fit well with the feel and direction of SDL?
* If you have used competitors with a similar feature, how do you feel that this proposal compares to those?
* How much effort did you put into your review? A glance, a quick reading, or an in-depth study?
Please state explicitly whether you believe that the proposal should be accepted into SDL.

More information about the SDL evolution process is available at

https://github.com/smartdevicelink/sdl_evolution/blob/master/process.md

Thank you,
Theresa Lech

Program Manager - Livio
theresa@livio.io

**Comment by E-SAITO-TMC** [comment]
I think it's good that notifications are automated.
But where does this proposal come from in the SDL process?
It can not be determined whether notification from SHAID or notification from SDL Server is better.
I think the notification system should be considered after the process decision.
I'm sorry if the SDL process is clearly documented.
Please tell me which process notification.

**Comment by nickschwab** [comment]
@E-SAITO-TMC Hello, this proposal suggests that the app review email notifications should be sent from an OEM's SDL Server installation. This is proposed as the best solution in order to avoid complications with OEM firewalls and GDPR constraints which would otherwise exist if the SDLC were to send the emails via the Developer Portal or SHAID.

**Comment by E-SAITO-TMC** [comment]
@nickschwab 
GDPR constraints are very difficult.
But first I would like to clarify the App review process.
App developers get UUID, embed SDL, apply for App Review to SDLC, and proceed to App Review for OEMs.
Is my understanding correct?

**Comment by nickschwab** [comment]
@E-SAITO-TMC Yes, that is correct. After an app has applied for review and is approved by the SDLC, the app is then distributed to OEM SDL Servers where each OEM has an opportunity to approve or deny the app from running in their vehicles.

This proposal is to allow an OEM to notify themselves via an internal email when a new application is ready for review on their SDL Server.

**Comment by joeygrover** [comment]
@E-SAITO-TMC the proposal itself is simply to allow an OEM to send an email to themselves using their email server when an app is ready for approval on THEIR SDL server. This means that even if the app certification process changes for the SDLC, at some point the app information will be sent  to an OEM's SDL Server and this feature would allow an email to be sent at this time.

App information can also be updated after they have been certified. For example, an app adds a new nickname or requests a new permission. While they do not need to go through the app certification process again, the OEM's SDL Server will be notified that this information changed and the OEM has the opportunity to approve those changes or additional permission requests. 

In short, even if the app certification process changes, this feature will still be necessary.

**Comment by theresalech** [comment]
The Steering Committee voted to defer this proposal, keeping it in review, to allow for further clarification and discussion of the proposal on the review issue.  The Steering Committee will vote on this proposal during the 2019-05-21 meeting.

**Comment by E-SAITO-TMC** [comment]
@nickschwab @joeygrover 
Thank you your comment.
I have checked the security policy with the network administrator.

**Comment by theresalech** [comment]
The Steering Committee fully agreed to accept this proposal.  It was noted that if members have additional concerns about this proposal, they should bring them to the Steering Committee discussion as soon as they arise.

**Comment by theresalech** [comment]
Implementation Issue: https://github.com/smartdevicelink/sdl_server/issues/128
